generic_tracers/generic_COBALT.F90

```diff
@@ -271,13 +265,12 @@ subroutine generic_COBALT_register(tracer_list)
       write (stdoutunit,*) trim(note_header), 'Simulating radiocarbon'
     endif
 
-    if (trim(co2_calc) == 'ocmip2') then
-      write (stdoutunit,*) trim(note_header), 'Using FMS OCMIP2 CO2 routine'
-    else if (trim(co2_calc) == 'mocsy') then
+    if (trim(co2_calc) == 'mocsy') then
```

Comment:
Do we need this moving forward, if we're not providing multiple options for the co2 calculation?

Reply:
Oh, I see there is already an issue open regarding this: [https://github.com/NOAA-CEFI-Regional-Ocean-Modeling/ocean_BGC/issues/71](url)

Reply:
@jessluo, thank you! Yes, this is the first step for #71, simplifying the current CO2 calculation module. We are keeping `co2_calc` in this PR but will consider removing it in the future if we offer only one option to users.